src/refract/Query.h

```diff
@@ -11,6 +11,7 @@
 #include "ElementFwd.h"
 
 #include <functional>
+#include <string>
```

Comment:
For some reason, without this I get a build error since std::string (used below) is not defined.

```
/Users/kyle/Projects/apiaryio/drafter/src/refract/Query.h:24:31: Implicit instantiation of undefined template 'std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
```

Not sure how this works for you? Unless functional in some compilers include string too.